### ISSUE TITLE
Go 1.15: conversion from int to string yields a string of one rune

### ISSUE THREAD

**Issue by eclipseo** [akamai/AkamaiOPEN-edgegrid-golang#101]
```
github.com/akamai/AkamaiOPEN-edgegrid-golang/configgtm-v1_3
# github.com/akamai/AkamaiOPEN-edgegrid-golang/configgtm-v1_3
./datacenter.go:213:59: conversion from int to string yields a string of one rune
./datacenter.go:263:59: conversion from int to string yields a string of one rune
FAIL	github.com/akamai/AkamaiOPEN-edgegrid-golang/configgtm-v1_3 [build failed]
error: Bad exit status from /var/tmp/rpm-tmp.BoK9mf (%check)
```
This issue is due too https://github.com/golang/go/issues/32479

**Comment by eclipseo** [comment]
Fixed in 1.0.0.

**Comment by zstlaw** [comment]
Thank you for following this up.  I was going to ask soon.
